log/aggregator.go

```diff
@@ -0,0 +1,18 @@
+package log
+
+import (
+	"time"
+)
+
+// Aggregator is an interface for pluggable components that collect log messages delivered via
+// some transport mechanism
+type Aggregator interface {
```

Comment:
The `logs` package is gone in this PR, so there might now be too much confusion after all.

In general, I really struggle sometimes with package and type name in Go. If this had been `aggregator.Aggregator`, for instance, I believe the lint checker complains about the "stuttering" type name. So I struggle to come up with `<package>.<type>` names that "read well" and make sense. (`storage.Adapter` was an example of that from the last time this was refactored.) I liked how `log.Aggregator` _sounded_. That is, it reads well, but you are correct that from a semantic point of view, it's not as informative as `transport.Aggregator`, but that doesn't read well. I always feel conflicted over this stuff...

I'll be happy to go with whatever you think is best.